### PR TITLE
Create joelschwartz.md

### PR DESCRIPTION
**Pull request by joelschwartz**
https://raw.githack.com/joelschwartz/joelschwartz.github.io/joelschwartzbranch/#!pages/vi/profiles/joelschwartz.md

### CODE REVIEW

**Comment by joelschwartz** [comment]
The directions do not clearly say what markdown elements are. I do not know anything about markdown elements. I know how to write HTML but the directions say not to use it.

**Comment by svlesiv** [comment]
Please take a look on official GitHub Guide for Markdown syntax: https://guides.github.com/features/mastering-markdown/

Also, you could go through this tutorial to learn Markdown: 
https://tylingsoft.github.io/tutorial.md/#whats-markdown

Let me know if you have more questions.

**Comment by lmmrssa** [comment]
@joelschwartz There is a link on Introductory page under **Step 3 - Markdown and Fork Tutorial** which describe what markdown is. Please read all the steps carefully.

File: pages/vi/profiles/joelschwartz.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+1. My name is Joel Schwartz
+2. I am from Mansfield Ohio 
+3. my time zone is eastern standard time
+![adirondack mountains view](https://www.wildernessphotographs.com/images/xl/Boreas-Mt-Treetop-crop-of-Elk-Lake-High-Peaks-_1000px.jpg)
```
Comment (svlesiv) [inline pages/vi/profiles/joelschwartz.md]: Looks much better. Awesome job!
However, the image and order number are shifted. To be able to fix that, you would need to add an empty line above the image.

Screen Shot 2019-03-25 at 11 17 14 AM

File: pages/vi/profiles/joelschwartz.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+1. My name is Joel Schwartz
+2. I am from Mansfield Ohio 
+3. my time zone is eastern standard time
+
+![adirondack mountains view](https://www.wildernessphotographs.com/images/xl/Boreas-Mt-Treetop-crop-of-Elk-Lake-High-Peaks-_1000px.jpg)
+
+* The operating system that I use is windows ten home 1803.
+* I am familiar with *HTML5*, *CSS3*,  **Javascript**, **SQL**, **C#**, *ASP.NET SPA framework*, and *PHP*.
```
Comment (lmmrssa) [inline pages/vi/profiles/joelschwartz.md]: You seem to have tried using multiple markdown. Which is good but having different markdown on same context does not look nice. Using markdown is part of this step but so is having good profile page of yours.
So can you make consistent markdown on list of language sets.

**Comment by lmmrssa** [comment]
@joelschwartz @dogi Closing this PR as it seems to have been open for long time and no update on it.

**Pull request closed without merging**